.travis.yml

```diff
@@ -29,4 +29,4 @@ branches:
 before_install:
   - if test "$TRAVIS_OS_NAME" == "osx"; then brew update; brew install openssl; ln -s /usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib /usr/local/lib/; ln -s /usr/local/opt/openssl/lib/libssl.1.0.0.dylib /usr/local/lib/; fi
 script:
-  - ./build.sh
+  - ./build.sh |tee /dev/null
```

Comment:
Mvc is feeling a bit defensive here 😺. It's `msbuild` and the problem (Microsoft/msbuild#1792) doesn't just affect Mvc.

`tee`ing is a viable workaround. But, we shouldn't check this in to a single .travis.yml file. My aspnet/KoreBuild#192 will do this everywhere. Yeah, kind-of like @Eilon suggested we might want to enable source link support.